src/modules/GroupChannelList/components/ChannelListHeader/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+
+import { LocalizationContext } from '../../../../lib/LocalizationContext';
+import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
+import Label, { LabelTypography, LabelColors } from '../../../../ui/Label';
+
+import './index.scss';
+import Avatar from '../../../../ui/Avatar';
+
+interface ChannelListHeaderInterface {
+  renderHeader?: (props: void) => React.ReactElement;
+  renderTitle?: () => React.ReactElement;
+  renderIconButton?: (props: void) => React.ReactElement;
+  onEdit?: (props: void) => void;
+  allowProfileEdit?: boolean;
+}
+
+const ChannelListHeader: React.FC<ChannelListHeaderInterface> = ({
+  renderHeader,
+  renderTitle,
+  renderIconButton,
+  onEdit,
+  allowProfileEdit,
+}: ChannelListHeaderInterface) => {
+  const { stores, config } = useSendbirdStateContext();
+  const { user } = stores.userStore;
+  const { logger } = config;
+
+  const { stringSet } = useContext(LocalizationContext);
+
+  if (renderHeader) {
+    logger?.warning('Recomend to use "renderTitle" instead of "renderHeader". It will be deprecated.');
+  }
```

Comment:
If `renderHeader` has been deprecated in ChannelList, it seems safe to remove it in GroupChannelList.

Reply:
Good catch!